libs/framework/src/dm_service_dependency.c

```diff
@@ -342,6 +342,22 @@ bool celix_dmServiceDependency_isTrackerOpen(celix_dm_service_dependency_t* depe
     return isOpen;
 }
 
+bool celix_dmServiceDependency_isSetCallbackConfigured(celix_dm_service_dependency_t* dependency) {
+    bool isUsed;
```

Comment:
This is old-style C convention. Is that still something Celix goes for? (no judgement meant here)

Reply:
this was unintentional. Updated this.